src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -261,7 +255,8 @@ trait AttributeComponent {
               value_number double DEFAULT NULL,
               value_boolean bit(1) DEFAULT NULL,
               value_entity_ref bigint(20) unsigned DEFAULT NULL,
-              list_index int(11) DEFAULT NULL, """
+              list_index int(11) DEFAULT NULL,
+              list_length int(11) DEFAULT NULL, """
```

Comment:
Sorry, my comment was misleading. I meant: would you rather use:
* DEFAULT NULL
or
* NOT NULL DEFAULT 0

In general, I avoid db columns that allow null whenever possible; that's the point here. Nulls can mean we forgot about the row. But this is a pretty small point.